### PR TITLE
[4.3] guided tours module count

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes

Changes the way tours are counted in the module. The result is the same but without the warning about unreachable code

### Testing Instructions
Check to make sure that there are a maximum of 7 tours in the module dropdown
Change the count in the module and check that the dropdown is changed correctly

Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by coolcat-creations** [comment]
I do not have a warning before applying the patch. How can I reproduce it? (Error Reporting is set to maximum)

**Comment by brianteeman** [comment]
Sorry for not being clear. The warning about unreachable code will only be seen in an ide. You just need to follow the test instructions to confirm that after the changed code the count still works

**Comment by coolcat-creations** [comment]
Thank you @brianteeman <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40447">issues.joomla.org/tracker/joomla-cms/40447</a>.</sub>

**Comment by Quy** [comment]
Another way is to add the following after line 55:
`$listTours = array_slice($listTours, 0, $params->get('tourscount', 7));`

and remove:
```
            <?php if ($i >= $params->get('tourscount', 7)) : ?>
                <?php break; ?>
            <?php endif; ?>
```

**Comment by brianteeman** [comment]
@quy - I am sure there are lots of ways

**Comment by HLeithner** [comment]
I created a pull request for you https://github.com/brianteeman/joomla-cms/pull/496

**Comment by brianteeman** [comment]
@HLeithner there are lots of unrelated changes in your pr

**Comment by HLeithner** [comment]
> @HLeithner there are lots of unrelated changes in your pr

only code style and revert to what it was before, but you can just pick what you need and commit it your self. I don't need to be mentioned.

**Comment by brianteeman** [comment]
I will look tomorrow. I don't use the computer on Saturday

File: administrator/modules/mod_guidedtours/tmpl/default.php [hunk 1]
```diff
@@ -25,16 +25,16 @@
     ->getWebAssetManager()
     ->useScript('bootstrap.dropdown');
 
-$lang = $app->getLanguage();
-
+$lang       = $app->getLanguage();
 $extension = $app->getInput()->get('option');
```
Comment (Quy) [inline administrator/modules/mod_guidedtours/tmpl/default.php]:
```suggestion
$extension  = $app->getInput()->get('option');
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 83c0a3077f23ea4150c182137b7f1d9c5a3e23c6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40447">issues.joomla.org/tracker/joomla-cms/40447</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40447">issues.joomla.org/tracker/joomla-cms/40447</a>.</sub>

**Comment by obuisard** [comment]
Thank you Brian @brianteeman for the PR!

**Pull request merged**